tests/vmi_configuration_test.go

```diff
@@ -3146,3 +3150,20 @@ func withSerialBIOS() libvmi.Option {
 		vmi.Spec.Domain.Firmware.Bootloader.BIOS.UseSerial = pointer.Bool(true)
 	}
 }
+
+func createBlockDataVolume(virtClient kubecli.KubevirtClient) (*cdiv1.DataVolume, error) {
+	sc, foundSC := libstorage.GetBlockStorageClass(k8sv1.ReadWriteOnce)
+	if !foundSC {
+		return nil, nil
+	}
+
+	dataVolume := libdv.NewDataVolume(
+		libdv.WithNamespace(util.NamespaceTestDefault),
```

Comment:
again, I think it does not need namespace here, and we should return the actual dv from line 3166, not the empty definition
`	dataVolume, err := virtClient.CdiClient().CdiV1beta1().DataVolumes(util.NamespaceTestDefault).Create(context.Background(), dataVolume, metav1.CreateOptions{})`